testdata/project-v3-with-deploy-image/controllers/busybox_controller.go

```diff
@@ -164,6 +201,17 @@ func (r *BusyboxReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		err = r.Update(ctx, found)
 		if err != nil {
 			log.Error(err, "Failed to update Deployment", "Deployment.Namespace", found.Namespace, "Deployment.Name", found.Name)
+
+			// The following implementation will update the status
+			busybox.SetConditionBusybox(metav1.Condition{Type: typeDegradedBusybox,
+				Status: "True", Reason: "Resizing",
```

Comment:
Could we use the const defined in the lib `metav1.ConditionTrue`?
same in the other places